code/modules/mod/modules/modules_maint.dm

```diff
@@ -182,6 +182,46 @@
 	mod.wearer.put_in_hands(balloon)
 	drain_power(use_power_cost)
 
+/// Paper dispenser - dispenses (sometimes burning) paper sheets
+/obj/item/mod/module/paper_dispenser
+	name = "MOD paper dispenser module"
+	desc = "A simple module designed by the bureaucrats of Torch Bay. \
+		It dispenses 'warm, clean, and crisp sheets of paper' onto a nearby table. Usually."
+	icon_state = "paper_maker"
+	module_type = MODULE_USABLE
+	complexity = 1
+	use_power_cost = DEFAULT_CELL_DRAIN * 0.5
+	incompatible_modules = list(/obj/item/mod/module/paper_dispenser)
+	cooldown_time = 5 SECONDS
+	/// The total number of sheets created by this MOD. The more sheets, them more likely they set on fire.
+	var/num_sheets_dispensed = 0
+
+/obj/item/mod/module/paper_dispenser/on_use()
+	. = ..()
+	if(!.)
+		return
+	if(!do_after(mod.wearer, 1 SECONDS))
+		return FALSE
+
+	var/obj/item/paper/crisp_paper = new(get_turf(src))
+	crisp_paper.desc = "It's crisp and warm to the touch. Must be fresh."
+
+	var/obj/structure/table/nearby_table = locate() in range(1, mod.wearer)
+	to_chat(mod.wearer, span_notice("You dispense a sheet of [crisp_paper.name] [nearby_table ? "onto [nearby_table]":"into your hands"]. It's warm, almost steaming."))
+	playsound(get_turf(src), 'sound/machines/click.ogg', 50, TRUE)
+
+	mod.wearer.put_in_hands(crisp_paper)
+	if(nearby_table)
+		mod.wearer.transferItemToLoc(crisp_paper, nearby_table.drop_location(), silent = FALSE)
+
+	// Up to a 30% chance per sheet to set on fire, 2% per sheet made
+	if(prob(min(num_sheets_dispensed * 2, 30)))
```

Comment:
```suggestion
	if(prob(min(num_sheets_dispensed * 2, 30)))
		mod.wearer.balloon_alert(mod.wearer, "PC LOAD LETTER")
```

Reply:
Isn't there literally an error code for this?